.gitignore

```diff
@@ -1,3 +1,7 @@
 node_modules
 .env
+Oauth.js
```

Comment:
@Gwillison415 I would recommend moving these files somewhere outside the repo. Please put them in a safe place and remove them from the repo, thanks!